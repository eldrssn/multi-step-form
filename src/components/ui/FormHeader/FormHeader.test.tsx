import { render } from '@testing-library/react';
import { FormHeader } from './';

test('displays correct text data', () => {
  const { getByText, container } = render(
    <FormHeader title="Title" description="Description" />
  );

  expect(getByText('Title')).toBeInTheDocument();
  expect(getByText('Description')).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
