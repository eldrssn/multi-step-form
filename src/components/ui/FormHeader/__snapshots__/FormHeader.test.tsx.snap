// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays correct text data 1`] = `
<div>
  <h1
    class="title"
  >
    Title
  </h1>
  <p
    class="description"
  >
    Description
  </p>
</div>
`;
